[docs]readable URLs (#7925)
ToolJet/ToolJet: docs/docs/dashboard.md

## Workspace Manager

The workspace manager is located on the bottom left corner of the dashboard. Clicking on the workspace manager will open a dropdown menu with a list of all the workspaces you are a part of. You can switch between workspaces by clicking on the workspace name from the dropdown menu.

<div style={{textAlign: 'center'}}>

<img className="screenshot-full" src="/img/dashboard/workspacemenu.png" alt="Dashboard"/>

</div>

### Add new workspace

On clicking the `Add new workspace` button, a modal will open where you can enter the name of the workspace, enter the unique workspace slug, and can see the preview of the workspace URL. Click on the `+ Create Workspace` button to create a new workspace.

> Previously, the workspace slug was the workspace id which was generated automatically. An example of the workspace slug was `https://tooljet.com/262750db-b2b8-4abb-9404-8995c2ecb2a0`. The new workspace slug is a unique name that you can choose for your workspace. An example of the new workspace slug is  `https://tooljet.com/apac-team `.

Reviewer: Previously, the workspace slug was an automatically generated workspace ID, like this: https://tooljet.com/262750db-b2b8-4abb-9404-8995c2ecb2a0'. Now, you can choose a custom, unique slug for your workspace (can add a ss here for reference) which will generate a more accessible or readable URL such as https://tooljet.com/apac-team."

Reviewer: also, please check if we can add a screenshot here - it will be good. if we are already adding here, then it will be good to highlight the specific areas / parts so that user gets it.

Author: ```suggestion
Previously, the workspace slug was an automatically generated workspace ID, like this: `https://tooljet.com/262750db-b2b8-4abb-9404-8995c2ecb2a0`. Now, you can set a custom, unique slug for your workspace which will generate a more accessible or readable URL such as `https://tooljet.com/apac-team)`.
```

Author: Made the changes and the screenshot is already there.

> Previously, the workspace slug was the workspace id which was generated automatically. An example of the workspace slug was `https://tooljet.com/262750db-b2b8-4abb-9404-8995c2ecb2a0`. The new workspace slug is a unique name that you can choose for your workspace. An example of the new workspace slug is  `https://tooljet.com/apac-team `.

Author: ```suggestion
Previously, the workspace slug was an automatically generated workspace ID, like this: `https://tooljet.com/262750db-b2b8-4abb-9404-8995c2ecb2a0`. Now, you can set a custom, unique slug for your workspace which will generate a more accessible or readable URL such as `https://tooljet.com/apac-team)`.
```

> If the table with the same name is already present in the workspace, the new table will be created with the name `<table name>_<unix timestamp>`. Example: `<tablename>_1627980000`.

Reviewer: what is this for?

Author: Ignore this section. That's not the part of readable url.

</div>

### Slug for folders

> The folder's URL slug is generated automatically from its name, providing direct access to the folder using the slug.

Author: ```suggestion
The folder's URL slug is generated automatically from its name, providing direct access to the folder using the slug.

To get the URL of a specific folder, the user will have to select that folder and then copy the URL from the address bar of the browser. The copied URL can be used to share with other users of the workspace.
```

> The slug for folders is automatically generated based on the name of the folder. The slug for folders can be used to access the folder directly from the URL. 

Reviewer: The folder's URL slug is generated automatically from its name, providing direct access to the folder using the slug.
@withshubh please change it to this.

> The slug for folders is automatically generated based on the name of the folder. The slug for folders can be used to access the folder directly from the URL. 

Author: ```suggestion
The folder's URL slug is generated automatically from its name, providing direct access to the folder using the slug.
```

> Example: If the name of the folder is `customer support`, The folder can be accessed directly from the URL `https://tooljet.com/<workspace-name>?folder=customer%20support`.

Reviewer: please add one more statement on how users can get this link.
- Basically user has to go to the folder to get the URL from the link in search bar of browser. Rephrase it better.
